starter_code/views/beers.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="beer-container">
+{{#each beers}} 
+{{ }}
```

Comment:
and here you can add a partial
```{{> beerPartial this}}```

remember that hen you have to register the partial